### PR TITLE
Tests/environment variables

### PR DESCRIPTION
**Pull request by mbadan**
## Description
Addition to integration tests regarding Environment Variables

### CODE REVIEW

File: packages/server/src/api/routes/tests/environmentVariables.spec.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      b: "number",
+    })
+    expect(pg.queryMock).toHaveBeenCalled()
+    expect(res.body.rows.length).toEqual(1)
+    expect(events.query.previewed).toBeCalledTimes(1)
+    expect(events.query.previewed).toBeCalledWith(
+      response.body.datasource,
+      query
+    )
+    //expect(postgres.integration).toHaveBeenCalledWith()
```
Comment (mike12345567) [inline packages/server/src/api/routes/tests/environmentVariables.spec.ts:137]: Little comment needs removed here!

File: packages/server/src/api/routes/tests/environmentVariables.spec.ts [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      .post(`/api/queries`)
+      .send(structures.basicQuery(response.body.datasource._id))
+      .set(config.defaultHeaders())
+      .expect("Content-Type", /json/)
+      .expect(200)
+    expect(response2.body._id).toBeDefined()
+  })
+
+  it("should run a query preview and check the mocked results", async () => {
+    // TODO: use the preview API
```
Comment (mike12345567) [inline packages/server/src/api/routes/tests/environmentVariables.spec.ts:94]: Think TODO is done, preview appears to be used.

**Review by mike12345567 — APPROVED** [review]
LGTM - just two comments to be removed!

File: packages/server/src/api/routes/tests/environmentVariables.spec.ts (hunk 1 above)
Comment (mbadan) [reply packages/server/src/api/routes/tests/environmentVariables.spec.ts]: got it removed!

File: packages/server/src/api/routes/tests/environmentVariables.spec.ts (hunk 2 above)
Comment (mbadan) [reply packages/server/src/api/routes/tests/environmentVariables.spec.ts]: got it removed!

**Review by shogunpurple — APPROVED** [review]
LGTM! Nice work 🎉

**Pull request merged**